index.html

```diff
@@ -77,7 +69,47 @@ <h4 class="sub-title">I am a Software Developer</p>
 
       </section>
 
+      <section class="wrap">
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        
+      <header>
+        <h1 class="heading">Multi-Purpose Stories</h1>
+     </header>  
+     
+     <main>
+    <section>
+        <div>
+    <p class="text">
+        A daily selection of privately 
+         personalized reads; no accounts or 
+          sign-ups required. has been the 
+           industry's standard dummy text ever 
+           since the 1500s, when an unknown 
+           printer took a standard dummy text.
+    </p>
+    </div>
+    
+    <nav class="button">
+        <button class="button1" type="css">css</button>
+        <button class="button2" type="html">html</button>
+        <button class="button3" type="bootstrap">bootstrap</button>
+        <button class="button4" type="Ruby">Ruby</button>
+        <button class="button5" type="See Project">See Project</button>
```

Comment:
- [ ] Having elements that load quickly while displaying a simple interface, consider adding button styling: Active, Hover, and Pressed to your button ✨. [see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)